packages/deployments/contracts/test/connext.spec.ts

```diff
@@ -1042,4 +1018,321 @@ describe("Connext", () => {
     );
     expect(testMessage).to.be.eq(serializedMessage);
   });
+
+  describe("multipath", () => {
+    const params = {
+      to: user.address,
+      callData: "0x",
+      originDomain,
+      destinationDomain,
+    };
+    const amount = 1000;
+    let message: any;
+    let nonce: any;
+    let transferId: any;
+    let bridgedAmount: any;
+    let reconciledTopics: any;
+
+    beforeEach(async () => {
+      await originTm.setupRouter(router2.address, router2.address, router2.address);
+      await originTm.setupRouter(router3.address, router3.address, router3.address);
+      await destinationTm.setupRouter(router2.address, router2.address, router2.address);
+      await destinationTm.setupRouter(router3.address, router3.address, router3.address);
```

Comment:
Lets make sure to merge #915 is merged first

Reply:
915 already merged